terraform/testcases/prometheus_static/otconfig.tpl

```diff
@@ -0,0 +1,23 @@
+receivers:
```

Comment:
will we be able to use it in ec2?

Reply:
It should be able to. Right now it isn't, it only works for EKS, as we need to add some environment variables (`SAMPLE_APP_HOST` and `SAMPLE_APP_PORT`) to the EC2 instance so the Collector can point to the right host.